src/style.css

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");

```

Comment:
change for this link, cause it imports not only 400, but also bolder font-weight
https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap

Reply:
use this for font-style `font-family: 'Inter', sans-serif;`

Reply:
sooo can I use it for the catalog or do I still have tweaks to do? :d